### DISCUSSION TITLE
I want to Test Deploy after `clasp push`

### DISCUSSION THREAD

**Discussion by AmonsLintao** [google/clasp#1006] [Q&A]
Is there a command that can trigger a `Test Deploy`?

I have looked into `deploy` command, seems like this is no stuff related to test deploy

**Comment by psqmerlo** [comment]
Hello
Did you figure out how to fix this? I'm having the same issue

**Comment by ilias777** [comment]
Hi. I have the same question.

The only way that I found at the moment is, that you go to your project with `clasp open` and press the deploy button and then "test deployment". From there keep the Web-App open in your browser and after clasp push, reload the page and your changes are applied.

I hope in the future comes a command to test a deployment.
